runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatConfig.java

```diff
@@ -82,6 +82,16 @@ default boolean theoretical()
 		return false;
 	}
 
+	@ConfigItem(
+		keyName = "equipmentToggle",
+		name = "Equipment Inventory Toggle",
```

Comment:
This should have a better name that gives a clearer function

Reply:
What about `equipmentStatsToggle`?